### DISCUSSION TITLE
Import Data CSV - Link Columns

### DISCUSSION THREAD

**Discussion by conorw** [Budibase/budibase#4806] [General]
Hi,
Is there any way to import linked data into a budibase internal database table?

I.e. I have 2x tables: Users & Attendance
I have imported users into the Users table using a CSV file.
I have a 2nd csv containing the attendance for each user (and their newly created user id as a column).
However, if I import the attendance data, it does not import the linked user information.
Is there any way to do this?
Thanks

**Comment by mjashanks** [comment]
Hey Conor. There is no way to do this today, other than via the API & a script.

**Reply by conorw** [reply]
thanks @mjashanks, would you have an example of a script to get me started?

**Reply by conorw** [reply]
Its OK @mjashanks, I was able to work it out using the browser Dev tools + Postman:
https://stackoverflow.com/questions/56448021/bulk-post-put-api-requests-using-postman-or-any-other-means

**Reply by ghost** [reply]
Hi @conorw 
Did you find a work around, I have the same issues. Currently I have to manually do it, its such a pain. Please let me know

**Reply by conorw** [reply]
@cheechbcn unfortunately the approach in this issue did not work out for me.
It created a strange situation in my database where there was old deleted data that was not being cleared from the database, so the approach is not recommended.
In the end, I gave up using relational data and just denormalized the data into each table, mainly for performance reasons.
Here is some related background info: https://github.com/Budibase/budibase/discussions/5299#discussioncomment-2530529
